provider/src/main/kotlin/org/gradle/kotlin/dsl/accessors/CodeGenerator.kt

```diff
@@ -16,71 +16,96 @@
 
 package org.gradle.kotlin.dsl.accessors
 
+import org.gradle.api.Project
+import org.gradle.api.internal.HasConvention
+import org.gradle.api.plugins.ExtensionAware
+
 import org.jetbrains.kotlin.lexer.KotlinLexer
 import org.jetbrains.kotlin.lexer.KtTokens
 
 
 internal
 fun ProjectSchema<TypeAccessibility>.forEachAccessor(action: (String) -> Unit) {
-    extensions.map(::typedAccessorSpec).forEach { spec ->
-        extensionAccessorFor(spec)?.let(action)
+    val seen = SeenAccessorSpecs()
+    extensions.mapNotNull(::typedAccessorSpec).forEach { spec ->
```

Comment:
:x: Why `mapNotNull`?

Reply:
Because of https://github.com/gradle/kotlin-dsl/pull/624/files#diff-07f871d01bc99de19d82b0b68db03332R325